force-app/main/default/lwc/rd2EntryFormScheduleSection/__tests__/rd2EntryFormScheduleSection.test.js

```diff
@@ -0,0 +1,343 @@
+
+            controller = new Rd2EntryFormScheduleSectionTestController(component);
+            getRecurringSettings.mockResolvedValue(mockGetRecurringSettings);
+            document.body.appendChild(component);
+            await setupWires();
+        });
+
+        afterEach(() => {
+            jest.useRealTimers();
+        });
+
+        it('number of planned installments is not rendered', async () => {
+            await flushPromises();
+            expect(controller.numberOfPlannedInstallments()).toBeNull();
+        });
+
+        it('when recurring type changed to fixed, number of planned installments is visible', async () => {
+            const recurringType = controller.recurringType();
+            changeValue(recurringType, 'Fixed');
+            await flushPromises();
+
+            expect(controller.numberOfPlannedInstallments()).toBeTruthy();
+        });
+
+        it('default installment period is set on load', async () => {
+           await flushPromises();
+           expect(controller.customPeriod().value).toBe('Monthly');
+        });
+
+        it('day of month picklist defaults to current date', async () => {
+            await flushPromises();
+            const today = new Date().getDate().toString();
+            const expectedDay = today === '31' ? 'Last_Day' : today;
+            expect(controller.dayOfMonth().value).toBe(expectedDay);
+        });
+
+        it('returnValues returns data from visible fields by field api name', async () => {
+            await flushPromises();
+            controller.recurringType().value = 'Open';
+            controller.startDate().value = TEST_DATE;
+            const values = component.returnValues();
+            expect(values).toMatchObject({
+                "RecurringType__c": "Open",
+                "StartDate__c": TEST_DATE,
+                "npe03__Installment_Period__c": "Monthly",
+            });
+        });
+
+        it('resetValues resets visible lightning-input-fields', async () => {
+            await flushPromises();
+            controller.recurringType().value = 'Open';
+            controller.startDate().value = TEST_DATE;
+            component.resetValues();
+            expect(mockReset).toHaveBeenCalledTimes(3)
+        });
+
+    });
+
+    describe('default recurring type fixed', () => {
+        beforeEach(async () => {
+            component = createElement('c-rd2-entry-form-schedule-section', { is: rd2EntryFormScheduleSection });
+            component.addEventListener('installmentschange', mockHandleInstallmentsChange);
+            controller = new Rd2EntryFormScheduleSectionTestController(component);
+            getRecurringSettings.mockResolvedValue({ ...mockGetRecurringSettings, defaultRecurringType: 'Fixed' });
+            document.body.appendChild(component);
+            await setupWires();
+        });
+
+        it('number of planned installments is not rendered', async () => {
```

Comment:
Should this description be "is rendered"?

Reply:
Same question as @lparrott.  Seems reversed...